packages/paste-core/components/form/src/select/OptionGroup.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {safelySpreadFormControlProps} from '../shared/Utils';
+
+export interface OptionGroupProps extends React.OptgroupHTMLAttributes<HTMLOptGroupElement> {
+  label: string;
+  children: React.ReactNode;
+}
+
+const OptionGroup = React.forwardRef<HTMLOptGroupElement, OptionGroupProps>(({label, children, ...props}, ref) => (
+  <optgroup ref={ref} label={label} {...safelySpreadFormControlProps(props)}>
```

Comment:
Also discovered you can give these _some_ styles, but limited 

```
const OptionGroupElement = styled.optgroup(
  css({
    background: 'transparent',
    color: 'colorText',
    fontFamily: 'fontFamilyText',
    fontWeight: 'fontWeightMedium',
    fontStyle: 'normal',
    margin: 'space20',
  })
);```

![image](https://user-images.githubusercontent.com/368249/80732992-f4497f00-8ac1-11ea-9525-72e30e537870.png)